tests/utilities_t/executors_t/test_sshexecutor.py

```diff
@@ -43,7 +43,7 @@ async def run(self, command, input=None, **kwargs):
             elif command != "Test":
                 raise ValueError(f"Unsupported mock command: {command}")
             return AttributeDict(
-                stdout=input and input.decode(), stderr="TestError", exit_status=0
+                stdout=input and input, stderr="TestError", exit_status=0
```

Comment:
`input and input` results in `input` if I am not mistaken. Can't the `and input` just be removed?

Reply:
Yes, you are right. I will change it.

Reply:
It looks like the recent commit only fixed one occurence (in `sshexecutor.py` itself) but not this one.
```suggestion
                stdout=input, stderr="TestError", exit_status=0
```